### PR TITLE
Adjust css for contactfolder

### PR DESCRIPTION
**Pull request by elioschmutz**
Adjust css for letters for ftw.theming:

Before:
test2

After:

test

### CODE REVIEW

**Comment by bierik** [comment]
@elioschmutz please update the whole ftw.contact to the new ftw.theming. Using the new variables set.

**Comment by elioschmutz** [comment]
@bierik :+1: updated the scss. It now uses the ftw.theming components.

File: ftw/contacts/resources/contacts.scss [hunk 1]
```diff
@@ -23,6 +24,9 @@ $resource-path: "#{$portal-url}/++resource++ftw.contacts.resources";
     padding: 1em 0px;
```
Comment (bierik) [inline ftw/contacts/resources/contacts.scss]: Use `padding: $padding-vertical 0`. Never use a custom value such as `1em`;

File: ftw/contacts/resources/contacts.scss [hunk 2]
```diff
@@ -23,6 +24,9 @@ $resource-path: "#{$portal-url}/++resource++ftw.contacts.resources";
...
     text-align: center;
     &:hover, &:focus, &:active, &.active {
```
Comment (bierik) [inline ftw/contacts/resources/contacts.scss]: :scissors: last &.active

File: ftw/contacts/resources/contacts.scss [hunk 3]
```diff
@@ -126,25 +128,25 @@ $resource-path: "#{$portal-url}/++resource++ftw.contacts.resources";
...
+        height: $portrait-height;
         background-size: contain;
     }
 
     .contactFolderLoadMoreContacts {
-        @include button($contact-letter-standalone-color);
+        @include button($color-button-default);
         margin-left: 30%;
         margin-right: 30%;
         width: 40%;
```
Comment (bierik) [inline ftw/contacts/resources/contacts.scss]: I think here you can use the floated grid from ftw.theming.

File: ftw/contacts/resources/contacts.scss [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+    $portrait-height);
+
 $resource-path: "#{$portal-url}/++resource++ftw.contacts.resources";
 
-@mixin letter($disabled) {
-    @include button($contact-letter-standalone-color, $disabled);
+@mixin letter($disabled: false) {
+    @include button-default($disabled: $disabled);
     margin: 0px;
     padding: 0px;
```
Comment (bierik) [inline ftw/contacts/resources/contacts.scss]: Use `0` because the unit doesn't matter when setting to none.

**Comment by bierik** [comment]
Some little changes but the rest looks :+1: :sparkles:

**Comment by elioschmutz** [comment]
@bierik :+1: Adjusted.

The &:active and &.active are both necessary. If a letter is selected, it gets the `active` class.

bildschirmfoto 2016-02-03 um 08 35 06

**Pull request merged**